api/services/locationService.ts

```diff
@@ -9,27 +9,27 @@ const log = getLogger('locationService');
 
 function mapToLocationDto(agency: ScosAgencyDto): LocationDto | null {
   // "Site" is an array of 1 element
-  const site = agency.site_info.site[0];
+  const [site] = agency.site_info.site;
 
   if (!site) {
     // there's no location data
     return null;
   }
 
   // as far as I can tell, "Address" has 2 elements but the second is a throwaway
-  const address = site.address[0];
+  const [address] = site.address;
   const address1 = address.line1;
   const address2 = `${address.line2} ${address.line3}`.trim();
 
   // not sure the difference between "st" vs "stg" phones
-  // so we'll just pick the first one that exists
+  // so we'll just show them all
   const phoneList = [...site.stgphones, ...site.stphones];
-  const phoneNumber = phoneList[0]?.phone ?? '';
+  const phoneNumber = phoneList.join(', ');
```

Comment:
the elements in `phoneList` are objects of the format:

```
{
  phone: string;
  type: string;
}
```
if you want to display them all, we at least need to do something like `phoneList.map(x => x.phone);` (and probably unique them) first. It might be better to return as an array though, and just update the UI.

I picked one site at random, and this is the list if we just comma-separate:
```
(614) 241-2569 Administrative, (614) 221-3554 Administrative St. Vincent de Paul, (614) 241-2531 JOIN, (614) 241-2569 Administrative
```
(not particularly useable, imo)

Reply:
Why don't you think that is usable? It gives them all the phone numbers and extensions if applicable. If a number is out of order and we don't display other ones that would be really frustrating.

Reply:
There are no line breaks, so it would make the location dialog potentially wider than the screen (or cut off the text, idk exactly how it's handled)

You're right that we should probably display all the numbers. I think ideally they'd each be on their own line.

Reply:
We can join them on "\n" then or "<br>" and that will put a line for each number. We could also send back an array but then we'd have to update the UI.